guides/schematics.md

```diff
@@ -1,26 +1,30 @@
 Angular Material comes packaged with Angular CLI schematics to make
 creating Material applications easier.
 
-## Install Schematics
-Schematics come packaged with Angular Material, so once you have
+### Install Schematics
+Schematics come packaged with Angular Material. Once you have
 installed the npm package, they will be available via the Angular CLI.
 
-Using the command below will automatically install Angular Material for you
+Using the command below will install Angular Material for you
```

Comment:
We should mention that the CDK will be also installed.

Reply:
Good call. Added along with some links to relevant guides.